styleguide/components/00-settings/_colors.scss

```diff
@@ -141,10 +144,21 @@ $themes: (
     'field-description-background': color('dark-gray', -6),
     'field-description-color': color('dark-gray'),
 
-    'field-color': color('dark-gray'),
-    'field-background': color('white'),
-    'field-border': 1px solid color('dark-gray', 3),
-    'field-placeholder': color('dark-gray', 3),
+    'field-color': color('dark-gray', 1),
+    'field-color--success': color('dark-gray', 1),
+    'field-color--error': color('red'),
+    'field-background': color('cyan', -6),
+    'field-background--focus': color('cyan', -5),
+    'field-background--disabled': color('dark-gray', -6),
+    'field-background--success': color('green', -6),
+    'field-background--error': color('red', -6),
+    'field-border': 1px solid color('dark-gray', -1),
+    'field-border--focus': 1px solid color('cyan'),
+    'field-border--disabled': 1px solid color('dark-gray', -3),
+    'field-border--success': 1px solid color('green'),
+    'field-border--error': 1px solid color('red'),
+    'field-placeholder': color('dark-gray', -1),
+
```

Comment:
That's a lot of variables already...
Can we really not define 'error-color' and then use it for all atoms which require an error color?

Imagine maintaining:
field-background--error, field-message-text--error, message-border--error, button-background--error, em--error...

You'll end up with 50 shades of red quite soon ;)

Reply:
This might indeed be a problem... but it has to be discussed further, because that will limit the configuration of components in possible sub themes...
I agree it's a lot of variable now, but I'm gonna keep thinking about a better solution